app/validators/claim/advocate_claim_sub_model_validator.rb

```diff
@@ -14,19 +15,14 @@ def has_one_association_names_for_steps
   def has_many_association_names_for_steps
     {
       case_details: [],
-      defendants: [
-        :defendants
-      ],
+      defendants: %i[defendants],
       offence_details: [],
-      fees: %i[
-        basic_fees
-        misc_fees
-        fixed_fees
-        expenses
-        messages
-        redeterminations
-        documents
-      ]
+      fixed_fees: %i[basic_fees fixed_fees],
```

Comment:
i would rename this to something more unequivocal - e.g. basic_or_fixed_fees, main_fees - because it gets confusing to look at a form step that is also an association